.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build and test
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  create:
+
+jobs:
+  build-and-test-code:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
This should be java 11 as that's our org-wide standard (and the runtime of the shipped images)

Reply:
Then we don't even need java installed. It comes with Java 11 out of the box